cmd/state/commands/opcode_tracer.go

```diff
@@ -396,7 +397,7 @@ type segPrefix struct {
 
 // OpcodeTracer re-executes historical transactions in read-only mode
 // and traces them at the opcode level
-func OpcodeTracer(genesis *types.Genesis, blockNum uint64, chaindata string, numBlocks uint64,
```

Comment:
Perhaps make `logger` the last argument

Reply:
Ok - will do - I'll commit again once I've made the edits.  I think there are several more in that package.